ws/test_ws_ping.py

```diff
@@ -0,0 +1,334 @@
+        validityEndInSeconds=10*365*24*60*60,
+        KEY_FILE="/tmp/key.pem",
+        CERT_FILE="/tmp/cert.pem"):
+    haproxy_pem = "/tmp/out.pem"
+    out_files = []
+    k = crypto.PKey()
+    k.generate_key(crypto.TYPE_RSA, 4096)
+    cert = crypto.X509()
+    cert.get_subject().C = countryName
+    cert.get_subject().ST = stateOrProvinceName
+    cert.get_subject().L = localityName
+    cert.get_subject().O = organizationName
+    cert.get_subject().OU = organizationUnitName
+    cert.get_subject().CN = commonName
+    cert.get_subject().emailAddress = emailAddress
+    cert.set_serial_number(serialNumber)
+    cert.gmtime_adj_notBefore(0)
+    cert.gmtime_adj_notAfter(validityEndInSeconds)
+    cert.set_issuer(cert.get_subject())
+    cert.set_pubkey(k)
+    cert.sign(k, 'sha512')
+    with open(CERT_FILE, "wt") as f:
+        f.write(crypto.dump_certificate(
+            crypto.FILETYPE_PEM, cert).decode("utf-8"))
+        out_files.append(CERT_FILE)
+    with open(KEY_FILE, "wt") as f:
+        f.write(crypto.dump_privatekey(crypto.FILETYPE_PEM, k).decode("utf-8"))
+        out_files.append(KEY_FILE)
+    filenames = [CERT_FILE, KEY_FILE]
+    with open('/tmp/out.pem', 'w') as outfile:
+        for fname in filenames:
+            with open(fname) as infile:
+                for line in infile:
+                    outfile.write(line)
+    out_files.append(haproxy_pem)
+    return out_files
+
+
+def remove_certs(cert_files_):
+    for cert in cert_files_:
+        os.remove(cert)
+
+
+class Ws_ping(tester.TempestaTest):
+
+    backends = []        
+
+    clients = []
+
+    tempesta = {
+        'config': TEMPESTA_CONFIG % "",
+    }
+
+    # Client
+
+    def run_test(self, port, n):
+        time.sleep(2.0)
+        asyncio.run(self.ws_ping_test(port, n))
+
+    async def ws_ping_test(self, port, n):
+        host = hostname
+        ping_message = "ping_test"
+        for i in range(n):
+            async with websockets.connect(f"ws://{host}:{port}") as websocket:
+                await websocket.send(ping_message)
+                await websocket.recv()
+
+    async def wss_ping_test(self, port, n):
+        host = hostname
+        ping_message = "ping_test"
+        ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+        ssl_context.load_verify_locations("/tmp/cert.pem")
+        for _ in range(n):
+            async with websockets.connect(f"wss://{host}:{port}",
+                                          ssl=ssl_context) as websocket:
+                await websocket.send(ping_message)
+                await websocket.recv()
+
+    async def run_stress(port, n):
+        print("run_stress")
+        host = hostname
+        ping_message = "ping_test"
+        for _ in range(n):
+            timeout_ = randint(9, 10)
+            async with websockets.connect(f"ws://{host}:{port}",
+                                          timeout=timeout_) as websocket:
+                await websocket.send(ping_message)
```

Comment:
https://github.com/tempesta-tech/tempesta/issues/881 describes 2 stress tests: one with Tempesta restarts and another with short living connections loading Tempesta connection schedulers - is this the second case? I appreciate a test comment for people, like me, who doesn't know the `websocket` library, describing whether each loop iteration reestablishes a new connection.

Also I didn't find usage of `threaded_stress()`, but it is confusing since it has name `threaded` and `async` inside - I believe `async` uses couroutines, not threads

Reply:
threaded_stress() - its artifact, so it isnt used, i already remove it
scheduler stress test - is not implemented yed